### DISCUSSION TITLE
Input nexus alignment file for morphological and molecular data

### DISCUSSION THREAD

**Discussion by NellyOrny** [iqtree/iqtree2#361] [Q&A]
Hello, 
Please I would like to generate a tree combining sub-alignments from a morphological and a molecular alignment .
To do so I have generated 3 files: Molecular.nex (for DNA alignment subset), Morphological.nex (for Morphological data) and chartset.nex (to define the chartset and the model of each partition) 
However, am still getting an error message stating "Morpho.nex not found". What command line should I used please? 
I need help. 
Please

**Comment by roblanf** [comment]
Can you please post the command line, and the output that you get?

**Reply by NellyOrny** [reply]
Thank you for replying. with some modification in the input file, the error message changed.
Here is the COMMAND LINE: 
iqtree2.exe -s Morpho_Mol_Conc_Taxa.nex -spp Morpho_mol_Conc_charset.nex -bb 10000
Here is the ERROR MESSAGE
ERROR: Error found at line 40, column 11 (file position 86658):
ERROR: Expecting ';' at the end of the MATRIX command; found HenMaur instead

"HenMaur" is one of the taxa in the input file. So I don't know what to do. it seems like my input file is not correct and that is the reason why the analysis stopped. Please can you provide an example showing how I should structure the data while dealing with morphological AND molecular data for combined analysis? All I found online and on Youtube are showing only various molecular data types.
